### PR TITLE
feat(FWS-59): Ability to edit profile page

### PR DESCRIPTION
**Pull request by Abdur-Siddiqui**
JIRA: link to jira ticket
https://champlainsaintlambert.atlassian.net/browse/FWS-59

What is the ticket about and why are we making this change?

This ticket is about being able to update a member's profile settings in the profile sidebar panel so that a member can view their account information and update their information if needed.

What are the various changes and what other modules do those changes affect?

-Adding an update feature, which allows a member to update their personal information
-4 types of testing
-Selenium Test
-backend for the update account
-Front end for the updateAccount

UI Change:
image
image

### CODE REVIEW

File: fitwsarah_frontend/src/views/authentication/Signup.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-//i have to fix sign up issue
+
+
 function AddMemberProfile() {
-    const { user, isAuthenticated, getAccessTokenSilently } = useAuth0();
+    const {user, isAuthenticated, getAccessTokenSilently} = useAuth0();
     const [accessToken, setAccessToken] = useState(null);
+    useEffect(() => {
+        if (isAuthenticated) {
+            localStorage.setItem("userData", JSON.stringify(user));
```
Comment (Eli293) [inline fitwsarah_frontend/src/views/authentication/Signup.js:13]: Where is localstorage from?

File: fitwsarah_frontend/src/views/authentication/Signup.js [hunk 2]
```diff
@@ -71,13 +89,18 @@ function AddMemberProfile() {
 
     useEffect(() => {
         if (accessToken) {
+            const storedUserData = localStorage.getItem("userData");
```
Comment (emilegirardGit) [inline fitwsarah_frontend/src/views/authentication/Signup.js:92]: what does localStorage is used for?

**Review by emilegirardGit — APPROVED** [review]

File: fitwsarah_frontend/src/views/authentication/Signup.js (hunk 1 above)
Comment (Abdur-Siddiqui) [reply fitwsarah_frontend/src/views/authentication/Signup.js:13]: So local storage stores user information on the browser, i did this to try and fix the sign up bug.

File: fitwsarah_frontend/src/views/authentication/Signup.js (hunk 2 above)
Comment (Abdur-Siddiqui) [reply fitwsarah_frontend/src/views/authentication/Signup.js:92]: It used for persistence to save user data through the web browser, it sort of acts as a session.

**Review by Eli293 — APPROVED** [review]
Looks good, nice job

**Pull request merged**